gcloud/bigtable/table.py

```diff
@@ -145,3 +147,41 @@ def delete(self):
         client = self._cluster._client
         # We expect a `._generated.empty_pb2.Empty`
         client._table_stub.DeleteTable(request_pb, client.timeout_seconds)
+
+    def sample_row_keys(self):
+        """Read a sample of row keys in the table.
+
+        The returned row keys will delimit contiguous sections of the table of
+        approximately equal size, which can be used to break up the data for
+        distributed tasks like mapreduces.
```

Comment:
The service [takes][1] a `SampleRowKeysRequest` and the [only field][2] in that message definition is `table_name`.

[1]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/5d84776473347513db0b6c44e4f8503837a0d9fa/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_service.proto#L44-L46
[2]: https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/5d84776473347513db0b6c44e4f8503837a0d9fa/bigtable-protos/src/main/proto/google/bigtable/v1/bigtable_service_messages.proto#L89-L93